Source/WTF/wtf/Vector.h

```diff
@@ -889,6 +889,7 @@ class Vector : private VectorBuffer<T, inlineCapacity, Malloc> {
     void appendUsingFunctor(size_t, const Functor&);
 
     void insert(size_t position, value_type&& value) { insert<value_type>(position, std::forward<value_type>(value)); }
+    void insertFill(size_t position, const T& value, size_t dataSize);
```

Comment:
Can this take a std::span<> instead?

(this would be a good candidate for a unit test in Tools/TestWebKitAPI/Tests/WTF/Vector.cpp)